py_neuromodulation/nm_plots.py

```diff
@@ -54,17 +53,18 @@ def plot_df_subjects(
     )
 
     if hue is not None:
-        n_hues = df[hue].nunique()
+        # n_hues = df[hue].nunique() # Never used, remove?
 
         handles, labels = ax.get_legend_handles_labels()
-        l = plt.legend(
-            handles[0:n_hues],
-            labels[0:n_hues],
-            bbox_to_anchor=(1.05, 1),
-            loc=2,
-            title=hue,
-            borderaxespad=0.0,
-        )
+        # Never used, remove?
```

Comment:
The legend is used, but not the `l = ` expression. So `l` can be removed

Reply:
Oh, true, what was i thinking there... been a long time since using pyplot sorry